backend/agent_runner/aws.py

```
@@ -0,0 +1,42 @@
+from agent_runner import AgentRunner, register_runner
+from store import update_job
+from datetime import datetime
+
+logger = logging.getLogger(__name__)
+
+class AwsKilocodeRunner(AgentRunner):
+    async def start(
+        self,
+        job: AgentJob,
+        plan: CodingPlan,
+        cluster: IssueCluster,
```

Ruff (0.14.8)

Unused method argument: `cluster`

(ARG002)